WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -240,6 +243,13 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
         }
     }
 
+    private void setSearchViewHintColor() {
```

Comment:
To verify this behavior you need an account that has a good amount of sites for the `SearchView` to become visible. If you want to show it without that then you change the menu item's visibility here https://github.com/wordpress-mobile/WordPress-Android/blob/939e025d86af2ef113c4054e6843182321ba141f/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java#L663

Reply:
Thanks. That's very thoughtful of you. 🙂

Reply:
I'm still getting a Text contrast warning. 🤔 

<img src="https://user-images.githubusercontent.com/198826/69586164-efc2ec80-0f9e-11ea-800e-e7d49374346c.png" width="320">

Reply:
Yes, I am not sure how this slipped me because I was changing the colors and running tests to get one that makes this ratio accurate which I think I did. Thanks. I put the right color now in 4837104